arm-datalake-analytics/job/2016-11-01/swagger/job.json

```diff
@@ -882,6 +1068,234 @@
       },
       "description": "The Data Lake Analytics job error details."
     },
+    "JobRelationshipProperties": {
+      "properties": {
+        "pipelineId": {
+          "type": "string",
+          "format": "uuid",
+          "description": "the job relationship pipeline identifier (a GUID)."
+        },
+        "pipelineName": {
+          "type": "string",
+          "maxLength": 260,
+          "description": "the friendly name of the job relationship pipeline, which does not need to be unique."
+        },
+        "pipelineUri": {
+          "type": "string",
+          "description": "the pipeline uri, unique, links to the originating service for this pipeline."
+        },
+        "runId": {
+          "type": "string",
+          "format": "uuid",
+          "description": "the run identifier (a GUID), unique identifier of the iteration of this pipeline."
+        },
+        "recurrenceId": {
+          "type": "string",
+          "format": "uuid",
```

Comment:
It hasn't been reviewed, I don't think, but these are Guids. Should they instead just be defined as strings even though they are in a GUID format?